cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1492,4 +1492,13 @@
      */
     public static final String UPDATE_ALLOWED = PREFIX + "update.allowed";
   }
+
+  /**
+   * Constants for streaming
```

Comment:
Since these constants are only used in the Spark runtime, I would suggest moving them to the `SparkRuntimeUtils` class.

Reply:
Done!